src/components/grid/grid-container/grid-container.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GridContainerStyle from './grid-container.style';
+import GridItem from '../grid-item';
+
+const GridContainer = (props) => {
+  const {
+    children
+  } = props;
+  return (
+    <GridContainerStyle>
+      {children}
+    </GridContainerStyle>
+  );
+};
+
+GridContainer.propTypes = {
+  /** Defines the Components to be rendered within the GridContainer. Requires a GridItem */
+  children: PropTypes.instanceOf(GridItem).isRequired
```

Comment:
This will only work if one `GridItem` is provided. You can see the error when looking at the [storybook](http://localhost:9001/iframe.html?id=design-system-grid--full).

> Failed prop type: Invalid prop `children` of type `Array` supplied to `GridContainer`, expected instance of `GridItem`.